asapdiscovery-ml/asapdiscovery/ml/trainer.py

```diff
@@ -22,7 +22,11 @@
     ModelConfigBase,
     ModelType,
     SchNetModelConfig,
+    ViSNetModelConfig,
 )
+
+# guard for visnet import. Refer MTENN issue #42
+from mtenn.conversion_utils.visnet import HAS_VISNET
 from pydantic import BaseModel, Extra, Field, ValidationError, validator
```

Comment:
The goal here is to give some feedback for users who installed `mtenn` from conda, without preventing them from using the other models. Appreciate advice on a better way to go about that